src/main/webapp/resources/js/components/PageLoadingIndicator.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
```

Comment:
Maybe we can just reuse the `ContentLoading` component and update it as need be instead of having 2 similar components

Reply:
Makes sense, updated in 9225af5cf70fa5658f9c10b64307c6b58061ae7c